deploy/examples/operator-openshift.yaml

```diff
@@ -570,6 +570,13 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # Enable read affinity for rbd volumes.
+  # CSI_ENABLE_READ_AFFINITY: "true"
```

Comment:
What would the behavior be if we set the default to true, but topology labels aren't found? Will that cause errors in ceph? If ceph will anyway fall back to a different OSD, I wonder if it makes sense to always enable the feature, and no need to even ask the user if they want it. I would think everyone would want this behavior by default.

Reply:
> What would the behavior be if we set the default to true, but topology labels aren't found? Will that cause errors in ceph? If ceph will anyway fall back to a different OSD, I wonder if it makes sense to always enable the feature, and no need to even ask the user if they want it. I would think everyone would want this behavior by default.

If no crush values/labels are found, ceph CSI will just not set the options. There will be no error. 
I can default this to true internally and expose this option just in case user wants to turn it off.